### ISSUE TITLE
about the training loss and test loss when training on my own dataset

### ISSUE THREAD

**Issue by QingyueMeng** [natanielruiz/deep-head-pose#75]
Hi, It is a absolutly great work. But when tried to train the resnet50 on my own dataset, I got the loss like following:

![image](https://user-images.githubusercontent.com/33691506/63213719-52a41c80-c142-11e9-9eef-ab7abcc69701.png)

The test loss is far bigger than the train loss. And the test loss looked like won't be declined. Is it normal? Or it had been overfit from the scratch.

Maybe some errors occured somewhere, but I cannot find it. Can you give me some advices?

thank you.

**Comment by QingyueMeng** [comment]
I filtered the dataset so set the angles between -99 and 102. But the loss of yaw is always far bigger than pitch and roll? I didn't change the code from your repo. Is it normal? Or I should do anything else?

**Comment by QingyueMeng** [comment]
And you could tell how about the loss curve like?

**Comment by jt19520** [comment]
Hello, have you solved the problem?I'm curious how did you train your data set and how did you label a custom dataset accurately? Now I want to predict the Euler angle of my custom dataset, but I don't know how to label it and load it
